book/website/collaboration/remote-collab/remote-collab-management.md

```diff
@@ -0,0 +1,5 @@
+# Managing Distributed Teams
+<!---Add a few sentences capturing the concepts explained below--->
+
+## Content
+<!---Add sub-heading--->
```

Comment:
How do you feel about putting a note box here that says something like the following?

> Uh-oh, this section hasn't been completed yet! 😱 
> Think you could contribute something here?
> Follow this link to get started! [link to relevant issue and/or contributing guidelines]

Reply:
Yeahhh! I am gonna try populating it in the collaboration cafe now.